v2/pkg/commands/build/build.go

```diff
@@ -151,6 +165,51 @@ func Build(options *Options) (string, error) {
 	return compileBinary, nil
 }
 
+func generateBindings(options *Options) error {
+	options.Logger.Print("  - Generating obfuscated bindings: ")
+	defer func() {
+		options.Logger.Println("Done.")
+	}()
+	// Run go mod tidy to ensure we're up-to-date
+	stdout, stderr, err := shell.RunCommand(".", "go", "mod", "tidy")
+	if err != nil {
+		options.Logger.Println(stdout)
+		options.Logger.Println(stderr)
+		return err
+	}
+
+	self := os.Args[0]
+	var env []string
+	if options.Obfuscate {
+		env = append(env, "WAILS_OBFUSCATE=true")
+	}
+
+	if len(options.UserTags) > 0 {
+		genModuleTags := lo.Without(options.UserTags, "desktop", "production", "debug")
```

Comment:
See also comment in the dev use case, about redundancy with the module command itself.